src/components/search/search.js

```diff
@@ -1,101 +1,94 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { SearchFacade } from './search-facade';
 import SiteSearchAPIConnector from '@elastic/search-ui-site-search-connector';
-import { SearchProvider, WithSearch } from '@elastic/react-search-ui';
-import SearchBox from './search-box';
+import debounce from 'debounce';
+import classnames from 'classnames';
+
+export default class Search extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      SearchProvider: undefined,
+      useModal: !this.props.disableModal || !this.props.resultsOnly
+    };
+  }
+
+  /* Wait to load the full Search component */
+  loadSearch = () => {
+    import(
```

Comment:
💡 It can be difficult to know when the SearchProvider loads (which is good 😎), while testing you could add a console.log statement in this function to verify when the facade gets swapped out.